.gitignore

```diff
@@ -68,7 +68,6 @@ typings/
 
 # Editor directories and files
 .idea
-.vscode
```

Comment:
This can be temporary - just to share my setup to work both on Client and Server with different settings.

Reply:
You don’t need to change the gitignore. You can just git add your changes.